### PR TITLE
Release 7.0.0-alpha.5 package-lock.json

### PR DESCRIPTION
**Pull request by swapnil-verma-gl**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [x] Other... Please describe: Release PR - Pushing missed package-lock.json

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/cli/package-lock.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
       "version": "7.0.0-alpha.5",
       "license": "Apache-2.0",
       "dependencies": {
-        "@alfresco/js-api": ">=7.8.0",
```
Comment (MichalKinas) [inline lib/cli/package-lock.json:12]: Let's rollback the change for the cli, here js-api should have older version as we had some issues with that in the past

File: package-lock.json [hunk 2]
```diff
@@ -366,229 +366,6 @@
         }
       }
     },
-    "node_modules/@angular-devkit/build-angular/node_modules/@esbuild/android-arm": {
```
Comment (MichalKinas) [inline package-lock.json:369]: Not sure why do you have any changes here but I assume it depends on operating system so I would roll those change back too

**Review by MichalKinas — CHANGES_REQUESTED** [review]

File: lib/cli/package-lock.json (hunk 1 above)
Comment (swapnil-verma-gl) [reply lib/cli/package-lock.json:12]: So should I revert the whole file to the original version? or just this mentioned version?

File: package-lock.json (hunk 2 above)
Comment (swapnil-verma-gl) [reply package-lock.json:369]: If we roll back the entire file, we would have the same package-lock.json as before, which causes the release job to fail

File: lib/cli/package-lock.json (hunk 1 above)
Comment (MichalKinas) [reply lib/cli/package-lock.json:12]: Whole file please

File: package-lock.json (hunk 2 above)
Comment (MichalKinas) [reply package-lock.json:369]: I doubt this part was responsible as the issue was with `Original error: Pruned lock file creation failed. The following package was not found in the root lock file: @alfresco/js-api@>=8.0.0-alpha.5` and this version was not changed across this file

**Comment by MichalKinas** [comment]
Okay I found the real issue - `"@alfresco/js-api": ">=8.0.0-alpha.5",` this line in `package.json` in `cli`

**Review by MichalKinas — APPROVED** [review]

**Pull request merged**